### PR TITLE
Add missing information on token_type_ids for roberta model

### LINKED ISSUE #19744: Documentation related to token_type_ids for RoBERTa needs to be updated
### System Info

This is about online documentation at https://huggingface.co/docs/transformers/v4.23.1/en/model_doc/roberta#transformers.RobertaModel.forward.token_type_ids

### Who can help?

_No response_

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

RoBERTa has no token type embeddings by default, while the documentation, indicates:

```
token_type_ids (torch.LongTensor of shape (batch_size, sequence_length), optional) — Segment token indices to indicate first and second portions of the inputs. Indices are selected in [0, 1]:

* 0 corresponds to a sentence A token,
* 1 corresponds to a sentence B token.
```

### Expected behavior

The documentation should reflect that the default RoBERTa model has no token type embeddings.

### PR DESCRIPTION
**Pull request by raghavanone**
# What does this PR do?

Add missing information on token_type_ids for roberta model

Fixes #19744

- [X ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [X ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?

## Who can review?

@sgugger

### CODE REVIEW

**Review by sgugger — COMMENTED** [review]
Thanks for the PR. Can you be a bit more explicit as the configuration will always be initialized with this parameter. Maybe it needs to be 2 for instance? Or any vlaue >= 2?

**Comment by raghavanone** [comment]
> Thanks for the PR. Can you be a bit more explicit as the configuration will always be initialized with this parameter. Maybe it needs to be 2 for instance? Or any vlaue >= 2?

Done

File: src/transformers/models/roberta/modeling_roberta.py [hunk 1]
```diff
@@ -649,11 +649,14 @@ def update_keys_to_ignore(self, config, del_keys_to_ignore):
...
             [What are attention masks?](../glossary#attention-mask)
         token_type_ids (`torch.LongTensor` of shape `({0})`, *optional*):
-            Segment token indices to indicate first and second portions of the inputs. Indices are selected in `[0,
-            1]`:
+            Segment token indices to indicate first and second portions of the inputs. Indices are selected in
+            `[0,1]`:
 
             - 0 corresponds to a *sentence A* token,
             - 1 corresponds to a *sentence B* token.
+            This parameter can only be used when RobertaEmbeddings is initialized with type_vocab_size
```
Comment (sgugger) [inline src/transformers/models/roberta/modeling_roberta.py:657]:
```suggestion
            This parameter can only be used when the model is initialized with `type_vocab_size`
```

File: src/transformers/models/roberta/modeling_roberta.py [hunk 2]
```diff
@@ -649,11 +649,14 @@ def update_keys_to_ignore(self, config, del_keys_to_ignore):
...
+            parameter with value >= 2.
+            All the value  in given should be always < type_vocab_size.
```
Comment (sgugger) [inline src/transformers/models/roberta/modeling_roberta.py:659]:
```suggestion
            All the value in this tensor should be < type_vocab_size.
```

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**